app.py

```diff
@@ -79,6 +80,15 @@ def refresh():
 
     return jsonify(f'The datasets updated successfully! The current number of available datasets is {len(masader)}.')
 
+@app.route('/datasets/<int:index>/issue')
+def report_card_issue(index: int):
+    if not (1 <= index <= len(masader)):
+        return jsonify(f'Dataset index is out of range, the index should be between 1 and {len(masader)}.'), 404
+
+    title = request.get_json()['title']
+    message = request.get_json()['message']
```

Comment:
It was better in the previous commit:

```
title = request.args.get('title', default="", type=str)
message = request.args.get('message', default="", type=str)
```

As it handles defaults.

Reply:
I have added a default value, didn't use the same structure since the passed data will be in JSON format